### DISCUSSION TITLE
Users role(s) should be displayed when clicking on their username

### DISCUSSION THREAD

**Discussion by weather7774** [stoatchat/discussions#778] [Feature Suggestions]
Basically as the title says.

This is an issue because when I click on a member with a colored name, I want to be able to understand why they have that special name. Whether it marks them as a valued member of some kind, or a moderator, etc.

**Comment by amycatgirl** [comment]
This is being worked on inside the upcoming client redesign, which you can test [here](https://revolt.chat/login)
